types/context.go

```diff
@@ -18,7 +18,7 @@ func NewContext(header abci.Header, isCheckTx bool, txBytes []byte) Context {
 	}
 	c = c.setBlockHeader(header)
 	c = c.setBlockHeight(header.Height)
-	c = c.setChainID(header.ChainId)
+	c = c.setChainID(header.ChainID)
```

Comment:
LOL. Blame protobuf. I don't know if we can force the name change, maybe gogo lets us do that too ...

Reply:
Oh haha you did it! #hero